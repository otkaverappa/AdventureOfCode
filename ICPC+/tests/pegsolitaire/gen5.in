100
###.o.###
....oo...
o........
..o......
###.o.###

###.o.###
o.o...o..
o......oo
.o.......
###...###

###.o.###
....o....
o.......o
o.......o
###...###

###.o.###
.........
......o..
o........
###.o.###

###...###
o.......o
..o.oo...
.....o...
###...###

###...###
.o...o.o.
...o..o..
o.....o..
###...###

###...###
.........
........o
.o.......
###...###

###..o###
ooo......
.o...o..o
o........
###...###

###.o.###
..o.o.oo.
..o....o.
.........
###..o###

###...###
.o.....o.
....o.oo.
....o....
###...###

###.o.###
o..oo....
..o......
...o..o..
###.o.###

###.o.###
o..o..oo.
...o.....
..o......
###...###

###.oo###
..o......
o.oo.o...
.........
###..o###

###o..###
.o...oooo
o....o...
.........
###...###

###...###
....o...o
.o.o.....
...oo....
###...###

###.oo###
.o..o.o..
o.......o
....o....
###...###

###...###
.o...o...
.....o...
........o
###...###

###.o.###
....oo...
.........
..o......
###..o###

###..o###
.o.......
......oo.
.o..oo.o.
###...###

###.o.###
ooo......
.........
..o....o.
###o..###

###o.o###
...o.....
oo.o.....
........o
###.o.###

###oo.###
..o.o.ooo
o........
.........
###...###

###...###
..o...oo.
...o..o..
.o.....o.
###o..###

###.o.###
.......o.
o........
....o.o..
###...###

###o..###
o.oo.o...
.....o...
........o
###...###

###...###
..o.o....
..o..oo.o
.......o.
###..o###

###..o###
.oooo.o.o
.....o...
.........
###...###

###..o###
oooo.....
.........
....o....
###..o###

###...###
o........
oo.......
o.....o..
###ooo###

###o.o###
....ooooo
o........
.........
###...###

###...###
.o..o.oo.
.....ooo.
o........
###...###

###...###
o.o......
o...o.oo.
.oo......
###...###

###..o###
.........
.o.o..ooo
.o.......
###o..###

###o..###
o.o...o..
o.o..o..o
.........
###...###

###o..###
...o.o..o
..o......
.....o.o.
###...###

###..o###
oo.o..o..
...o.....
...o.....
###...###

###o..###
.o.o...o.
...o.....
..oo.....
###...###

###oo.###
...o.....
.....o...
o...oo...
###.o.###

###.oo###
.oo..o...
.........
.....o..o
###...###

###...###
.....oo..
..o......
.........
###.o.###

###.o.###
.......o.
...o.....
.o.......
###..o###

###..o###
o.....oo.
o.o....o.
o........
###...###

###..o###
o........
....o....
....oo...
###..o###

###...###
......o..
o.....oo.
...o.....
###.oo###

###.o.###
.oooo....
....o...o
.........
###...###

###...###
....o...o
.......o.
...o.....
###...###

###..o###
....ooo..
.ooo..o..
.........
###...###

###o..###
..o......
.o..o....
..o.o...o
###...###

###.o.###
.........
.oo.o....
....o.o.o
###...###

###..o###
...oo....
.....oo..
..o......
###...###

###...###
.o.o...o.
....oo...
...o.o...
###o..###

###o..###
.......o.
..ooo.o..
......o.o
###...###

###...###
o.o...o..
.o..o....
..o......
###.o.###

###oo.###
...o..o.o
....o.o..
o........
###...###

###..o###
......o.o
ooo..o..o
.........
###...###

###...###
..o..o..o
......o..
......o..
###...###

###...###
o.oo..o..
.........
.........
###...###

###..o###
o.o.....o
.o.o...oo
.........
###...###

###o..###
......o..
o....o..o
..o.....o
###..o###

###o..###
......o.o
.o...o.o.
o.o......
###...###

###...###
...o...o.
o......o.
.....o...
###...###

###...###
....oo...
oo.......
......o.o
###...###

###.oo###
.........
...o..o..
o........
###.o.###

###.o.###
...o...o.
....o...o
....ooo..
###...###

###...###
..o......
..o.oo...
...oooo..
###...###

###..o###
oo.....oo
o.......o
.........
###...###

###...###
......o..
.........
.o.......
###..o###

###.o.###
.....oo..
o..o.o.o.
o........
###...###

###.o.###
......o..
o........
.........
###...###

###..o###
.o....o..
..oo.....
......o..
###...###

###...###
...o.....
....o....
o..ooo...
###...###

###..o###
..o.oo.o.
....o....
....o.o..
###...###

###...###
.o.o..o.o
o......o.
.o....o..
###...###

###...###
o.ooo..oo
.oo......
.........
###...###

###...###
..o......
o..o.o...
.o.......
###.o.###

###.o.###
o..o.o...
.........
o.ooo....
###...###

###...###
.........
o......oo
o..o.....
###...###

###...###
oo.......
..o.o...o
o........
###...###

###o..###
o...o....
....o...o
.....o..o
###.o.###

###.o.###
oo......o
oo.o.....
......o..
###...###

###...###
....o....
..o....oo
.....ooo.
###...###

###.oo###
.o.......
o...o..o.
...o..o..
###...###

###o..###
..o.o....
.......o.
.o..o.o..
###...###

###..o###
...o...oo
....o.o..
.o..o....
###...###

###oo.###
..oo.o...
.oo..o...
.........
###...###

###...###
....o....
.........
o......o.
###.oo###

###...###
o.....oo.
.....o.o.
...o..o.o
###...###

###...###
.o..o..o.
.........
.....o...
###.o.###

###...###
...o.....
.........
..o.o.o..
###.o.###

###...###
....o...o
o.o..o..o
.....oo..
###...###

###...###
........o
....oo..o
.........
###...###

###o..###
.oo...o..
.o.o.o..o
.........
###...###

###...###
oo..o..o.
.........
..o.....o
###..o###

###o..###
o..o.oo..
.....o.o.
......o..
###...###

###...###
.o...o...
.ooo.....
.........
###...###

###...###
...oo.o..
oo..o....
oo.......
###...###

###.o.###
.........
..oo....o
......o..
###o.o###

###...###
..o......
.o.o.o.o.
o..o.....
###...###

###...###
.oo....o.
....o.o..
..o....o.
###o..###

###...###
o.oo...oo
.........
.o.oo....
###...###
