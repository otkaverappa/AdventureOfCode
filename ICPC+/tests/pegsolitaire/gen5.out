5 1
4 4
3 3
4 0
4 2
5 2
2 0
6 2
4 4
4 2
4 4
3 4
4 4
5 3
4 2
6 2
3 1
4 1
3 5
6 1
4 4
4 4
6 2
5 0
4 3
5 3
3 5
5 2
7 1
4 4
3 5
2 6
6 2
6 2
5 2
2 5
4 3
2 6
4 3
3 1
5 0
6 2
4 2
4 3
3 4
4 0
3 5
6 1
3 4
2 4
4 4
4 4
7 0
5 3
6 2
2 3
2 2
7 1
7 1
8 0
4 1
4 2
5 1
3 5
2 6
5 2
3 0
3 5
3 0
4 2
4 2
2 6
8 0
3 5
6 0
6 2
3 2
4 2
5 3
5 3
5 2
7 1
7 0
3 5
2 6
4 1
4 4
5 0
4 1
5 3
2 2
6 2
6 1
3 5
2 3
2 6
6 1
3 4
7 1
3 5
